### PR TITLE
Build angular_components with angular 6.0.0-dev

### PR DESCRIPTION
**Pull request by lukaszkolodziejczyk**

### CODE REVIEW

**Comment by kevmoo** [comment]
Wanna try using `dependency_overrides` to github to get travis to pass?

See example here

https://github.com/dart-lang/test/blob/38a132dc8fb8c0b47e577758652a93ba314e8cd1/pkgs/test/pubspec.yaml#L45-L91

File: examples/angular_components_example/pubspec.yaml [hunk 1]
```diff
@@ -69,3 +69,23 @@ dev_dependencies:
...
+      url: git://github.com/dart-lang/angular.git
+      path: angular_compiler
+  angular_router:
+    git:
+      url: git://github.com/dart-lang/angular.git
+      path: angular_router
+  angular_forms:
+    git:
+      url: git://github.com/dart-lang/angular.git
+      path: angular_forms
```
Comment (kevmoo) [inline examples/angular_components_example/pubspec.yaml:91]: Add missing trailing newline, as is tradition

```suggestion
      path: angular_forms

```

File: angular_gallery_section/pubspec.yaml [hunk 2]
```diff
@@ -19,3 +19,15 @@ dependencies:
...
+      url: git://github.com/dart-lang/angular.git
+      path: angular
+  angular_ast:
+    git:
+      url: git://github.com/dart-lang/angular.git
+      path: angular_ast
+  angular_compiler:
+    git:
+      url: git://github.com/dart-lang/angular.git
+      path: angular_compiler
```
Comment (kevmoo) [inline angular_gallery_section/pubspec.yaml:33]:
```suggestion
      path: angular_compiler

```

File: angular_gallery/pubspec.yaml [hunk 3]
```diff
@@ -14,3 +14,19 @@ dependencies:
...
+      url: git://github.com/dart-lang/angular.git
+      path: angular_ast
+  angular_compiler:
+    git:
+      url: git://github.com/dart-lang/angular.git
+      path: angular_compiler
+  angular_forms:
+    git:
+      url: git://github.com/dart-lang/angular.git
+      path: angular_forms
```
Comment (kevmoo) [inline angular_gallery/pubspec.yaml:32]:
```suggestion
      path: angular_forms

```

File: angular_components/pubspec.yaml [hunk 4]
```diff
@@ -30,3 +30,21 @@ dev_dependencies:
...
+      url: git://github.com/dart-lang/angular.git
+      path: angular_ast
+  angular_compiler:
+    git:
+      url: git://github.com/dart-lang/angular.git
+      path: angular_compiler
+  angular_forms:
+    git:
+      url: git://github.com/dart-lang/angular.git
+      path: angular_forms
```
Comment (kevmoo) [inline angular_components/pubspec.yaml:50]:
```suggestion
      path: angular_forms

```

**Comment by lukaszkolodziejczyk** [comment]
Tradition respected. :)

**Pull request merged**